pkg/model/stack.go

```diff
@@ -221,7 +221,7 @@ func GetStack(name, stackPath string, isCompose bool) (*Stack, error) {
 			svc.Build.Dockerfile = ""
 		} else {
 			svc.Build.Context = loadAbsPath(stackDir, svc.Build.Context)
-			svc.Build.Dockerfile = loadAbsPath(stackDir, svc.Build.Dockerfile)
+			svc.Build.Dockerfile = loadAbsPath(svc.Build.Context, svc.Build.Dockerfile)
```

Comment:
Does docker compose work if you do:
```
    build:
      context: ./folder
      dockerfile: folder/Dockerfile
```

instead of?
```
    build:
      context: ./folder
      dockerfile: Dockerfile
 ```

Reply:
no, what docker does is concatenate docker file to context.
If you run your first example you will receive an error like:
```
could not find /Users/xxx/Projects/context-dockerfile/folder/folder: stat /Users/xxx/context-dockerfile/folder/folder: no such file or directory
```